committeeoversightapp/views.py

```diff
@@ -118,32 +118,23 @@ def post(self, request, **kwargs):
                 new_document.save()
 
                 archived_transcript_url = archive_url(transcript_url)
-                print(archived_transcript_url)
 
+                extensions = {'.pdf': 'application/pdf', '.htm': 'text/html', '.html': 'text/html'}
                 ext = get_ext(transcript_url)
-                if ext.lower() == '.pdf':
-                    new_document_link = EventDocumentLink(
-                                            url=transcript_url,
-                                            document=new_document,
-                                            media_type="application/pdf"
-                                        )
-                    new_archived_document_link = EventDocumentLink(
-                                            url=archived_transcript_url,
-                                            document=new_document,
-                                            media_type="application/pdf"
-                                        )
-                elif ext.lower() == '.htm' or ext == '.html':
+
+                try:
```

Comment:
Since the CharField returns an empty string, you can just do:

```
media_type = extensions.get(ext.lower(), '')
```

Then, you can remove the Try-Except business.

Reply:
Much better, thanks!